### PR TITLE
consolidate sql tests into one

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
This is to reduce redundancy in tests.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
azure_rm_sqlserver_facts
azure_rm_sqldatabase
azure_rm_sqlfirewallrule

##### ANSIBLE VERSION
2.7

### CODE REVIEW

File: test/integration/targets/azure_rm_sqlserver/aliases [hunk 1]
```diff
@@ -1,3 +1,6 @@
 cloud/azure
 destructive
 shippable/azure/group4
+azure_rm_sqlserver_facts
+azure_rm_sqldatabase
```
Comment (yungezz) [inline test/integration/targets/azure_rm_sqlserver/aliases:5]: sqldatabase_facts?

File: test/integration/targets/azure_rm_sqlserver/tasks/main.yml [hunk 2]
```diff
@@ -45,10 +45,257 @@
       - output.changed == false
       - output.state == 'Ready'
 
+# azure_rm_sqlserver_facts tests
+
+- name: Create SQL Server - 2
```
Comment (yungezz) [inline test/integration/targets/azure_rm_sqlserver/tasks/main.yml]: i think it's unnecessary to create 2 service instance just to assert  facts return list

File: test/integration/targets/azure_rm_sqlserver/aliases (hunk 1 above)
Comment (zikalino) [reply test/integration/targets/azure_rm_sqlserver/aliases:5]: not yet merged, but we could add in advance?

File: test/integration/targets/azure_rm_sqlserver/tasks/main.yml (hunk 2 above)
Comment (zikalino) [reply test/integration/targets/azure_rm_sqlserver/tasks/main.yml]: ok, I can remove

File: test/integration/targets/azure_rm_sqlserver/aliases (hunk 1 above)
Comment (yungezz) [reply test/integration/targets/azure_rm_sqlserver/aliases:5]: oh, ok, i  thought it was there. no need to add it now in case some file not exists  error.

**Review by yungezz — APPROVED** [review]

**Pull request merged**